scripts/smoke-packaged-app.mjs

```
@@ -0,0 +1,278 @@
+			fail(
+				`the renderer invokes '${channel}' but the packaged preload does not permit it.\n` +
+					`  Permitted: ${[...allowed].join(', ')}\n` +
+					'  These ship as one artifact, so they must agree.'
+			);
+		}
+	}
+	console.log(`smoke: renderer invokes [${[...invoked].join(', ')}] — all permitted by the preload`);
+}
+
+async function findPageTarget(deadline) {
+	while (Date.now() < deadline) {
```

ESLint

[error] Replace `↹` with `··`

(prettier/prettier)